### PR TITLE
Refactoring company

### PR DESCRIPTION
**Pull request by Tsaikkari**
Added JobPostForm, JobPostList, AddJobPost component, JobPost and EditJobPost pages. 
I imported Navbar-logout component and added link to add job post component. 
companyProfile

jobposts

editJobPostPage

### CODE REVIEW

File: src/components/CompanyProfileForm/index.tsx [hunk 1]
```diff
@@ -1,135 +1,113 @@
...
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value, name } = e.target
+
+    setFormData((prevValue: any) => {
+      return {
+        ...prevValue,
+        [name]: value,
+      }
+    })
+  }
```
Comment (Wyfy0107) [inline src/components/CompanyProfileForm/index.tsx:33]: This is 💯

File: src/redux/actions/employer.ts [hunk 2]
```diff
@@ -20,7 +20,9 @@ export const registerEmployerRequest = (
   }
 }
 
-export const registerEmployerSuccess = (employerInfo: Credential) => {
+export const registerEmployerSuccess = (
+  employerInfo: CredentialStateEmployer
```
Comment (Wyfy0107) [inline src/redux/actions/employer.ts:28]: Why change the type here, I think `Credential` type was correct

**Review by Wyfy0107 — COMMENTED** [review]
Code looks good. However, about the design, you are showing create post form in the same page with post lists. Can we do it similar to the figma design, where you will only show the form when click on `create new` button

File: src/redux/actions/employer.ts (hunk 2 above)
Comment (Tsaikkari) [reply src/redux/actions/employer.ts:28]: Oh, yeah. But I've changed the registerEmployer recently in the update-employer branch anyway. :)

**Comment by Tsaikkari** [comment]
Oh, I thought you asked me to put them inside the same tab. I'll make a new for the create post.

**Review by Wyfy0107 — APPROVED** [review]
💯

**Pull request merged**